server/settings.py

```diff
@@ -69,29 +69,29 @@
 
 WSGI_APPLICATION = 'server.wsgi.application'
 
+# DATABASES = {
+#     'test': {
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+#     }
+# }
+
+# if 'TRAVIS' not in os.environ:
 DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': config('db_name'),
```

Comment:
This causes error in travis environment since config variables are not accessible.